LiskHQ/lisk
# Extend version format to allow development QA - Closes #1918

### What was the problem?
Development test round has the same version as a production release. Creating few test rounds and tagging them with the same version of development and production branches is confusing.
### How did I fix it?
Allowed version extension, ex: allow version extension form `1.0.0-beta.7` to `1.0.0-beta.7.0`
### How to test it?
Change the `version` and `minVersion` in `config.json` with the extended version `1.0.0-beta.7.0`, it should allow the node to start. you can also run the tests to verify the version extension.
### Review checklist

* The PR solves #1918 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
